controllers/productController.js

```diff
@@ -0,0 +1,20 @@
+const productService = require("../services/productService");
+
+const showDetail = async (req, res) => {
```

Comment:
함수명은 해당 함수가 어떤 행위를 하는지, 어떤 결과를 반환하는지 예측할 수 있도록 작성해주는 것이 좋습니다.

현재 각 레이어로 분리되었지만 큰 흐름을 고려한다면 상세페이지에서 특정 상품에 대한 id 값을 받아 상품 정보를 조회하는 것이기 때문에 아래와 같이 수정할 수 있겠습니다:
```suggestion
const getProductById = async (req, res) => {
```